pages/bench.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+import { Footer } from "/components/Footer";
+
+export default function Bench({ bench, setBench }) {
+  const benchMapper = () => {
+    if (bench.length === 0) {
+      return <Empty>It's empty here!</Empty>;
+    } else {
+      return bench.map((benchItem) => {
+        return <BenchDiv>{benchItem.charname}</BenchDiv>;
+      });
+    }
+  };
```

Comment:
You could do this with an inline ternary operator: `bench.length === 0 ? <Empty>…</Empty> : bench.map(…)` to get rid of the benchMapper function.

I personally do not like the if-inside-a-function-pattern, but it works I guess. ;)